aip/general/0008.md

```diff
@@ -168,9 +168,12 @@ described in [RFC 2119][].
 When using these terms in AIPs, they **must** be lower-case and **bold**. These
 terms **should not** be used in other ways.
 
-**Important:** If an appendix is used, it exists to provide background and a
-more complete understanding, but **must not** contain guidance (and RFC-2119
-terms **must not** be used).
+If "SHOULD" or "SHOULD NOT" are used, they **must** include valid examples of
+where the guidance can be ignored.
```

Comment:
instead of "ignored", how about "overridden by other concerns"?

Reply:
yeah, the wording is a bit unfair. Fixed, PTAL.